pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -362,7 +377,11 @@ func (s *DiscoveryServer) authenticationz(w http.ResponseWriter, req *http.Reque
 			info.ClientProtocol = authProtocolToString(clientProtocol)
 
 			if (clientProtocol & serverProtocol) == 0 {
-				info.TLSConflictStatus = "CONFLICT"
+				if clientProtocol == authnCustomMTls && serverProtocol != authnHTTP {
+					info.TLSConflictStatus = "?"
```

Comment:
Nit: Should we make this clear that it actually depends on user's mTLS configuration? In other words it's "OK" from the Istio's configuration perspective.
A question mark seems it's indicating something went wrong with the istioctl command.

Reply:
Change to MAY CONFLICT